src/Google.Api.Gax/IPagedAsyncEnumerable.cs

```diff
@@ -0,0 +1,20 @@
+﻿/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ * Use of this source code is governed by a BSD-style
+ * license that can be found in the LICENSE file or at
+ * https://developers.google.com/open-source/licenses/bsd
+ */
+
+using System.Collections.Generic;
+
+namespace Google.Api.Gax
+{
+    public interface IPagedAsyncEnumerable<T> : IAsyncEnumerable<T>
+    {
+        new IPagedAsyncEnumerator<T> GetEnumerator();
+    }
+
+    public interface IPagedAsyncEnumerable<TResponse, TResource> : IPagedAsyncEnumerable<TResponse>
```

Comment:
When we document this, we should explain why it exists at all. (Basically for type inference, IIRC.)